pinot-query-planner/src/main/java/org/apache/pinot/query/QueryEnvironment.java

```diff
@@ -158,12 +160,11 @@ public QueryEnvironment(TypeFactory typeFactory, CalciteSchema rootSchema, Worke
   public QueryPlannerResult planQuery(String sqlQuery, SqlNodeAndOptions sqlNodeAndOptions, long requestId) {
     try (PlannerContext plannerContext = new PlannerContext(_config, _catalogReader, _typeFactory, _hepProgram)) {
       plannerContext.setOptions(sqlNodeAndOptions.getOptions());
-      RelRoot relRoot = compileQuery(sqlNodeAndOptions.getSqlNode(), plannerContext);
-      SubPlan subPlanRoot = toSubPlan(relRoot);
+      RelRoot relRoot = planQueryLogical(sqlNodeAndOptions.getSqlNode(), plannerContext);
```

Comment:
since we use SqlPhysicalExplain, w no longer need the renamings, please revert back to `compileQuery` and `toDispatchableSubPlan`